components/calendar/events.md

```diff
@@ -76,6 +78,72 @@ When handling the `ViewChanged` event, you cannot use two-way binding for the `V
 
 >tip You are not required to provide an initial value to the `View` parameter. It will default to `CalendarView.Month`.
 
+## RangeStartChanged
+
+The `RangeStartChanged` fires every time the user selects a new starting date for the range of dates. The first click on a date in the Calendar will always be the starting date and the range can be consisted of dates only onwards.
```

Comment:
On a calendar with range selection only, perhaps? If so, clarify this in the sentence

Reply:
Also, you can select earlier dates, but they are considered a start of the range too

Reply:
The `RangeEndChanged` fires immediately after the start event with a weird date. Sync with dev and if this is expected, document it

Reply:
It fires with the `default` value, and that makes sense. See the range picker docs I made

Reply:
What i meant by *consisted of dates only onward.* is that the dates will be from the StartRange to the EndRange so onward from Start if that makes sense.

The rest of the comments are done with: 6517076ff1c6a5890590a43c1e44753654ca32e0.
Fixed a typo in: 2e3165f3ef7a59cf09397f57bff82ee98318fe07